gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started-kubernetes.adoc

```diff
@@ -0,0 +1,650 @@
+//TODO: POINT TO PROD REQUIREMENTS ONCE THEY ARE AVAILABLE. 
+//Create story
+
+* At least 1 node.
+
+* Support for the creation of a `Service` resource of the `LoadBalancer` type.
++
+To avoid deployment errors in this guide, your cluster must support this process. However, note that Flex Gateway supports all of the following Ingress Controller services: `ClusterIP`, `NodePort`, `LoadBalancer` (default), `ExternalName`. 
+
+* Compute resources needed for the deployment procedure are _very unlikely_ to exceed the following values (equivalent to the EKS t3.micro instance type): 
+
+** vCPU: 2 vCPUs
+** Memory: 1 GiB
+** Network: Up to 5 Gigabit
+** Max ENI: 2
+** Max IPs: 4 
+
+To determine whether your cluster is ready for the <<deploy-flex, deployment procedure>>, it is important to review <<check-cluster>> _before starting_ the deployment. The checks require you to run a set of `kubectl` commands.
+
+[[install-flex]]
+== Step 1: Download the Flex Gateway Image
+
+Use Docker to download the Flex Gateway image from the Docker Hub registry.
+
+. Check that Docker is running by executing the following command in a terminal window:
++
+[source,docker,subs=attributes+]
+----
+docker info
+----
++
+If you receive an error after executing this command, see the information on <<docker-troubleshooting,Docker troubleshooting>>.  
+. Run the following command in a terminal window:
++
+[source,docker,subs=attributes+]
+----
+docker pull mulesoft/flex-gateway:{gateway-ver-var}
+----
++
+When successful, this command prints a message similar to this one in your terminal window:
++
+----
+latest: Pulling from mulesoft/flex-gateway
+Digest: sha256:e55555abcdefg1234567zxynwo33333fadjf
+Status: Image is up to date for mulesoft/flex-gateway:latest
+docker.io/mulesoft/flex-gateway:latest
+----
++
+Use `sudo` if you encounter permission issues when running this command. 
+
+[[register-flex]]
+== Step 2: Register Flex Gateway
+
+Register the Flex Gateway image from a terminal window using a temporary token generated by Anypoint Runtime Manager. This procedure produces a local `registration.yaml` file that contains registration properties for a named gateway. These properties are incorporated into a Helm chart in <<deploy-flex>>.
+
+. From a terminal window, prepare a directory for a Flex Gateway registration file (`registration.yaml`).
+.. Create a directory:
++
+[source,mkdir]
+----
+mkdir flex-registration
+----
+.. Navigate to the new directory:
++
+[source,cd]
+----
+cd flex-registration
+----
+
+. Log in to https://anypoint.mulesoft.com/login/[Anypoint Platform^] and select *Runtime Manager*.
+. From Runtime Manager, locate the procedure for registering Flex Gateway:
+.. Click *Flex Gateways* in the side navigation panel.
+.. Click *Add Gateway*.
+.. Select *Kubernetes*.
+.. In the Kubernetes page, locate the command in the *Register your gateway* section:
```

Comment:
I think Kim preferred "On" the page.

Reply:
done

Reply:
Her guidance sometimes seemed arbitrary, but if that's what we're all doing for consistency... whatever. no big deal one way or the other